lib/test.py

```diff
@@ -222,6 +224,9 @@ def run(self, server):
         short_status = None
         result_checksum = None
 
+        if not os.path.exists(self.var_suite_path):
+            os.mkdir(self.var_suite_path)
```

Comment:
1. There is `safe_makedirs()` helper in lib/utils.py.
2. It is better to don't produce empty directories for all test suites, but rather create them on demand: right before `shutil.copy(self.tmp_result, self.reject)`.

Reply:
I agree with these points.

Reply:
Right, corrected.